runtime/src/lib.rs

```diff
@@ -1292,19 +1203,14 @@ construct_runtime! {
         Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 29,
         Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 30,
         Bounties: pallet_bounties::{Pallet, Call, Storage, Event<T>} = 31,
-        Lottery: pallet_lottery::{Pallet, Call, Storage, Event<T>} = 32,
-        Gilt: pallet_gilt::{Pallet, Call, Storage, Event<T>, Config} = 33,
         // Pallets
         OrmlVesting: orml_vesting::{Pallet, Storage, Call, Event<T>, Config<T>} = 34,
         Currencies: orml_currencies::{Pallet, Call, Event<T>} = 35,
         Tokens: orml_tokens::{Pallet, Storage, Event<T>, Config<T>} = 36,
         PolkadexFungibleAsset: polkadex_fungible_assets::{Pallet, Call, Storage, Event<T>} = 37,
         SubstrateeRegistry: pallet_substratee_registry::{Pallet, Call, Storage, Event<T>} = 38,
         PolkadexOcex: polkadex_ocex::{Pallet, Call, Storage, Config<T>, Event<T>} = 39,
-        TokenFaucet: token_faucet_pallet::{Pallet, Call, Event<T>, Storage, ValidateUnsigned} = 40,
         ChainBridge: chainbridge::{Pallet, Call, Storage, Event<T>} = 41,
```

Comment:
Please remove Chainbridge too

Reply:
Yes I was planning that in a separate ticket, but I will do it here